addons/medical_engine/script_macros_medical.hpp

```diff
@@ -73,7 +73,7 @@
 #define DAMAGE_TOTAL_COLORS 10
 
 // --- pain
-#define PAIN_UNCONSCIOUS EGVAR(medical,const_painUnconscious)
+#define PAIN_UNCONSCIOUS EGVAR(medical,painUnconsciousThreshold)
 #define PAIN_UNCONSCIOUS_DEFAULT 0.5
```

Comment:
I don't know if we need this after adding CBA setting.
https://github.com/acemod/ACE3/blob/039af66a8f8ef253870c61ae53d800d5a0f7cc13/addons/medical_engine/XEH_preInit.sqf#L17